### PR TITLE
Corrected typos in Readme

### PR DESCRIPTION
**Pull request by fredwbaker**
Corrected some typos in the readme file and cleaned up a few small phrasing issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,16 +209,16 @@ The best way to install this extension is to follow the instructions below. Howe
...
-- Use this to select a means of emphasization using bold variations or underline variations. 
+- Use this to select a means of creating emphasis using bold variations or underline variations. 
 
 #### Line height buttons
-- Use these buttons to increase or decrease line height to strain and improve the confort of reading.
+- Use these buttons to increase or decrease line height to strain and improve the comfort of reading.
 
 #### Always on/off button
-- Use this button to controls the default behaviour which is if words on pages are or aren't emphasized when loaded by default.
+- Use this button to controls the default behavior which is if words on pages are or aren't emphasized when loaded by default.
```
Comment (asieduernest12) [inline README.md:221]: British spelling here, sharp eye really, thank you

File: README.md [hunk 2]
```diff
@@ -343,11 +343,11 @@ In a digital world dominated by shallow forms of reading, we aim to encourage a
 
 ## Development
    1. Clone the project
-   2. Open in VS Code or your favourite editor
+   2. Open in VS Code or your favorite editor
```
Comment (asieduernest12) [inline README.md:346]: still British spelling

File: README.md (hunk 1 above)
Comment (fredwbaker) [reply README.md:221]: My 1st PR. Happy to do some more or make some adjustments as needed

**Comment by fredwbaker** [comment]
Thanks! I like the tool. It's very helpful.

**Pull request merged**